omod/src/main/webapp/pages/peru/analysisRequests.gsp

```diff
@@ -39,6 +66,18 @@
                 <td>
                    ${ encounter.orders ? encounter.orders.size() : "" }
                 </td>
+                <td>
+                    ${ ui.format(encounter.encounterType)}
+                </td>
+                <td>
+                    ${ ui.format(encounter.location)}
+                </td>
+                <td>
+                    ${ ui.format(encounter.creator)}
+                </td>
+                <td>
+                    <a href="http://localhost:8080/openmrs/htmlformentryui/htmlform/editHtmlFormWithStandardUi.page?patientId=${encounter.patient.uuid}&encounterId=${encounter.uuid}&definitionUiResource=file:configuration/pih/htmlforms/sampleCollection.xml">Ingresar orden</a>
+                </td>
```

Comment:
You can use the ui "pagelink" utility to create a page link with the proper relative path.  It will also generate the various request parameter for you, you can see this example:

https://github.com/PIH/openmrs-module-pihcore/blob/master/omod/src/main/webapp/pages/meds/drugOrders.gsp#L44